controllers/register.controller.js

```diff
@@ -0,0 +1,42 @@
+const UserModel = require('../models/user.model');
+
+module.exports = async (req, res) => {
+  try {
+    const user = new UserModel(req.body);
+    const savedUser = await user.save();
```

Comment:
So this can remain as is ?

Reply:
Apologies, this was hidden with 3 other conversations, I didn't see it. Yes it can remain as is, part of the review is to share knowledge that's why I also gave feedback on things that could be done another way.
I'll resolve this